doc/ref_arch/openstack/chapters/chapter01.md

```diff
@@ -33,9 +33,8 @@ Several NFV use cases are documented in OpenStack. For more examples and details
 
 <a name="1.3"></a>
 ## 1.3 Terminology
-Unless specific OpenStack terms are used, the terminology can be found [CNTT Reference Model Terminology](../../../ref_model/chapters/glossary.md#1.1).  
+The general terminology, except for this Reference ARchitecture specific terms, can be found [CNTT Reference Model Terminology](../../../ref_model/chapters/glossary.md#1.1); this Reference Architecture terms will be defined where used. 
```

Comment:
This currently doesn't read very clearly.
I assume you are saying
"General terminology definitions can be found [CNTT Reference Model] and specific terms relating to this reference architecture are to be found [here]" right?

Reply:
@iangardner22 Changed - Thanks